scripts/misc/generate_population_table.pl

```diff
@@ -131,7 +131,7 @@
 
   # loop over databases
   while (my ($dbname) = $sth->fetchrow_array) {
-    next if ($dbname !~ /^[a-z]+_[a-z]+_$db_type\_$e_version\_\d+$/i);
+    next if ($dbname !~ /^[a-z_]+_[a-z]+_$db_type\_$e_version\_\d+$/i);
```

Comment:
This will not skip databases of format _parta_partb_variation_101_38

Reply:
It matches for example `_homo_sapiens_variation_100_38`, is there a different example I'm missing?

Reply:
Databases that start with underscores are matched and not skipped.